app/src/test/java/com/x8bit/bitwarden/data/platform/repository/SettingsRepositoryTest.kt

```diff
@@ -1098,6 +1100,30 @@ class SettingsRepositoryTest {
             fakeAuthDiskSource.userState = MOCK_USER_STATE
             assertFalse(settingsRepository.isAuthenticatorSyncEnabled)
         }
+
+    @Test
+    fun `isAuthenticatorSyncEnabled set should generate symmetric key if key is null`() {
```

Comment:
Shouldn't this just be part of the existing `isAuthenticatorSyncEnabled` test?

Reply:
Fair point! I'll circle back and consolidate.